### DISCUSSION TITLE
Clarification on datasource permissions by default in free version

### DISCUSSION THREAD

**Discussion by dcarrion87** [grafana/grafana#59360] [Data Sources]
On this page: https://grafana.com/docs/grafana/latest/datasources/datasource_permissions/

> By default, data sources in an organization can be queried by any user in that organization. For example, a user with the Viewer role can issue any possible query to a data source, not just queries that exist on dashboards they have access to.

However, it doesn't indicate what mechanism allows them to do this? E.g. A viewer by default doesn't have access to explore/browse. Is it via the API? Specifically detailing the mechanism of how a user can circumvent this in the free version would be good.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
